src/execution/lineage_manager.cpp

```diff
@@ -389,6 +449,29 @@ void ManageLineage::Persist(PhysicalOperator *op, shared_ptr<LineageContext> lin
     }
     break;
   }
+  case PhysicalOperatorType::INDEX_JOIN: {
+    // CREATE TABLE INDEX_JOIN (lhs_value INT, rhs_value BIGINT, in_index INT, out_chunk_id INT)
+    LineageOpBinary *lop = dynamic_cast<LineageOpBinary *>(lineage->GetLineageOp(op->id, 0).get());
+    if (!lop) return;
+    TableCatalogEntry * table = Catalog::GetCatalog(context).GetEntry<TableCatalogEntry>(context,  DEFAULT_SCHEMA, tablename);
```

Comment:
Nit: move * next to table, so `TableCatalogEntry * table` -> `TableCatalogEntry *table`